cmd/lakefs/cmd/import.go

```diff
@@ -131,6 +133,16 @@ var importCmd = &cobra.Command{
 			Cataloger:          cataloger,
 			KeyPrefixes:        prefixes,
 		}
```

Comment:
```suggestion
			KeyPrefixes:        prefixes,
			Rocks:              isRocks,
		}
```

Reply:
Also, IMO - move the `if` to before the creation of the config, and put the entryCatalog in a varialbe - so that you'll already create the config with all the right values, without editing it later.

Reply:
Done.